test/unit/helpers/wsApi.js

```
@@ -13,188 +13,111 @@ var wsApi = require('../../../helpers/wsApi');
 var failureCodes = require('../../../api/ws/rpc/failureCodes');
 
```

I would like to group in context each parameter.

---

it would be repeated with text description then: ` 
version
    should return an error when version is float
`

---

I'm saying something like:
```
version
      should return error when using float

---

done